
#ifndef ENTITY_HPP
#define ENTITY_HPP

#include "entity/player.pb.h"
#include "entity/form.pb.h"
#include "entity/buffer.pb.h"
#include "entity/law.pb.h"
#include "entity/item.pb.h"
#include "entity/sect.pb.h"
#include "entity/stage.pb.h"
#include "entity/stheno.pb.h"
#include "entity/announce.pb.h"
#include "entity/task.pb.h"
#include "entity/educate.pb.h"
#include "entity/attach.pb.h"
#include "entity/pvp.pb.h"
#include "entity/specshop.pb.h"
#include "entity/short.pb.h"
#include "entity/chakra.pb.h"
#include "entity/worship.pb.h"
#include "entity/mail.pb.h"
#include "entity/eggs.pb.h"
#include "entity/condor.pb.h"
#include "entity/boss.pb.h"
#include "entity/answer.pb.h"
#include "entity/addiction.pb.h"
#include "entity/gift.pb.h"
#include "entity/vip.pb.h"
#include "entity/multicopy.pb.h"
#include "entity/achievement.pb.h"
#include "entity/practice.pb.h"
#include "entity/guild.pb.h"
#include "entity/dailypoints.pb.h"
#include "entity/escort.pb.h"
#include "entity/tower.pb.h"
#include "entity/tavern.pb.h"
#include "entity/guildbattle.pb.h"
#include "entity/wife.pb.h"
#include "entity/openact.pb.h"
#include "entity/jianzhong.pb.h"
#include "entity/pair.pb.h"
#endif